src/components/homepage/Hero.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { FunctionComponent } from "react";
+import clsx from "clsx";
+import styles from "./Hero.module.css";
+import Link from "@docusaurus/Link";
+
+const heroData = {
+  title: (
+    <>
+      A&nbsp;unified way to manage
+      <br />
+      applications and infrastructure
+    </>
+  ),
+  subtitle: (
+    <>
+      Vendor-agnostic, crowdsourced expertise, packaged
```

Comment:
I think this actually sounds a bit better: "Vendor-agnostic, packaged, ready-to-use crowdsourced expertise."

Reply:
🤔 I think it looks and sounds heavier as there are many adjectives separated by comma. IMHO with the current form it's easier to read and digest. For now I will keep it as it is, maybe @lukaszo as a second reviewer will also give feedback on that.

Reply:
Crowdsourced expertise is the noun and has to be at the end. Otherwise it doesn't sound right. Alternatively it could be: "Vendor-agnostic. Packaged. Crowdsourced expertise."

Reply:
I vote for version from @platten

Reply:
I gave it some thought, and I came to the conclusion that for now we should get rid of `packaged` word, as it can be misleading. For example, in cnab they have an option to package which means you can create a bundle with installation manifests + application itself so they support airgapped installation ootb. Currently, we cannot achieve such thing, we have only installation manifests which describe how to install sth but sources e.g. terraform modules, helm chart, docker images etc. are not in our manifests, and we cannot create such bundle, and AFAIK it's not in our near future scope to change that.

maybe `"Vendor-agnostic. Crowdsourced expertise."` is enough

Reply:
I like the shorter form`"Vendor-agnostic. Crowdsourced expertise."`. What do you think @lukaszo @platten?